.config/redis.config.php

```diff
@@ -5,7 +5,7 @@
     'memcache.locking' => '\OC\Memcache\Redis',
     'redis' => array(
       'host' => getenv('REDIS_HOST'),
-      'password' => (string) getenv('REDIS_HOST_PASSWORD'),
+      'password' => getenv('REDIS_HOST_PASSWORD_FILE') && file_exists(getenv('REDIS_HOST_PASSWORD_FILE')) ? trim(file_get_contents(getenv('REDIS_HOST_PASSWORD_FILE'))) : (string) getenv('REDIS_HOST_PASSWORD'),
```

Comment:
If the variable is specified, but the file doesn't exist, this will fall back to a likely non-existent `REDIS_HOST_PASSWORD` -- shouldn't it error out or something instead?

Reply:
Sorry, what I'm suggesting is that perhaps `file_exists(getenv('REDIS_HOST_PASSWORD_FILE'))` should be dropped from the expression so that `file_get_contents` will fail and give the operator at least a warning somewhere so they have some clue why they specified the environment variable for pulling from a file but it isn't working.

Reply:
> An [E_WARNING](https://www.php.net/manual/en/errorfunc.constants.php#constant.e-warning) level error is generated if filename cannot be found, length is less than zero, or if seeking to the specified offset in the stream fails.
>
> When file_get_contents() is called on a directory, an [E_WARNING](https://www.php.net/manual/en/errorfunc.constants.php#constant.e-warning) level error is generated on Windows, and as of PHP 7.4 on other operating systems as well.

(from https://www.php.net/file_get_contents)